### PR TITLE
doc: remove proto in rfc, link to proto file in codebase

### PR DESCRIPTION
**Pull request by xiang90**
Let's remove the duplicated proto in rfc. We now just link the proto to the proto file in codebase.

It has been a maintainer burden.

### CODE REVIEW

File: Documentation/rfc/v3api.proto [hunk 1]
```diff
@@ -1,290 +0,0 @@
...
-  ResponseHeader header = 1;
-  repeated storagepb.KeyValue kvs = 2;
-  // more indicates if there are more keys to return in the requested range.
-  bool more = 3;
-}
-
-message PutRequest {
-  bytes key = 1;
-  bytes value = 2;
-  int64 lease = 3;
```
Comment (heyitsanthony) [inline Documentation/rfc/v3api.proto]: https://github.com/coreos/etcd/blob/master/etcdserver/etcdserverpb/rpc.proto#L90 is missing the lease field

Comment (xiang90) [inline Documentation/rfc/v3api.proto]: thanks for double checking! Fixed.

**Comment by heyitsanthony** [comment]
lgtm so long as the code files are current with the deleted spec

**Comment by xiang90** [comment]
@heyitsanthony I will do a double checking again before merging this.

**Comment by xiang90** [comment]
double checked. Merging

**Comment by gyuho** [comment]
Lgtm thanks!

**Pull request merged**